trin-core/src/types/accumulator.rs

```diff
@@ -51,9 +52,12 @@ impl MasterAccumulator {
     pub fn try_from_file(master_acc_path: PathBuf) -> anyhow::Result<MasterAccumulator> {
         let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
         path.push(master_acc_path);
-        let raw = std::fs::read(path)?;
-        MasterAccumulator::from_ssz_bytes(&raw)
-            .map_err(|msg| anyhow!("Unable to decode master accumulator: {:?}", msg))
+        let raw = std::fs::read(&path)
+            .map_err(|_| anyhow!("Unable to find master accumulator at path: {:?}", path))?;
+        let master_acc = MasterAccumulator::from_ssz_bytes(&raw)
+            .map_err(|msg| anyhow!("Unable to decode master accumulator: {:?}", msg));
+        info!("Loaded master accumulator from: {:?}", path);
```

Comment:
We ought to move this log to the call-site of this function. The function may return an error, so the "loaded..." `info!` log may mislead the reader. We can log with `info!` or `error!` at the call-site based on the result.